packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -186,6 +188,34 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
+    public getRenderedCellIcon(
+        rowIndex: number,
+        columnIndex: number
+    ): Icon | null {
```

Comment:
This can't actually return `null`, can it? You throw an error if the icon can't be found.

```suggestion
    ): Icon {
```